compat/src/suspense.js

```diff
@@ -59,70 +59,36 @@ Suspense.prototype = new Component();
  * @param {Promise} promise The thrown promise
  */
 Suspense.prototype._childDidSuspend = function(promise) {
-	// saves 5B
-	const _this = this;
 
-	_this._suspensions.push(promise);
-	let len = _this._suspensions.length;
+	/** @type {import('./internal').SuspenseComponent} */
+	const c = this;
```

Comment:
Went with `c` here instead of `_this` cuz it matches `diff` a little more. We use `c` to refer to the currently rendering component in `diff` so I though aligning the name here would make it more consistent.